axelrod/strategies/dbs.py

```diff
@@ -0,0 +1,390 @@
+from axelrod.actions import Actions, Action
+from axelrod.player import Player
+
+
+C, D = Actions.C, Actions.D
+
+
+class DBS(Player):
+    """
+    Desired Belief Strategy as described in [Au2006]_
+    http://www.cs.utexas.edu/%7Echiu/papers/Au06NoisyIPD.pdf
+
+    A strategy that learns the opponent's strategy, and uses symbolic 
+    noise detection for detecting whether anomalies in player’s behavior
+    are deliberate or accidental, hence increasing performance in noisy 
+    tournaments.  
+
+    From the learned opponent's strategy, a tree search is used to
+    choose the best move
+
+    Default values for the parameters are the suggested values in the
+    article. When noise increases you can try to diminish 
+    violation_threshold and rejection_threshold
+
+    Parameters
+    ----------
```

Comment:
The tests are failing because we don't quite use numpy notation **exactly** for strategies. In this case we need to remove the heading. So not:

```
Parameters
----------------
```

but simply 

```
Parameters
```